parent/pom.xml

```diff
@@ -41,10 +41,8 @@
         <ignite-spring-boot-autoconfigure-ext.version>1.1.0-SNAPSHOT</ignite-spring-boot-autoconfigure-ext.version>
         <ignite-spring-boot-thin-client-autoconfigure-ext.version>1.1.0-SNAPSHOT</ignite-spring-boot-thin-client-autoconfigure-ext.version>
         <ignite-spring-cache-ext.version>1.1.0-SNAPSHOT</ignite-spring-cache-ext.version>
-        <ignite-spring-data-2.0-ext.version>1.1.0-SNAPSHOT</ignite-spring-data-2.0-ext.version>
-        <ignite-spring-data-2.2-ext.version>1.1.0-SNAPSHOT</ignite-spring-data-2.2-ext.version>
-        <ignite-spring-data-commons.version>1.2.0-SNAPSHOT</ignite-spring-data-commons.version>
-        <ignite-spring-data-ext.version>1.1.0-SNAPSHOT</ignite-spring-data-ext.version>
+        <ignite-spring-data-ext.version>2.0.0-SNAPSHOT</ignite-spring-data-ext.version>
+        <ignite-spring-data-commons.version>2.0.0-SNAPSHOT</ignite-spring-data-commons.version>
```

Comment:
`ignite-spring-data-commons` version should not be changed. This module was not changed.

Reply:
This is why I asked you about exactly what versions needed to be changed.

Ok, reverted `ignite-spring-data-commons` version change. Now, only `ignite-spring-data-ext` version gets changed. Is this right now?

Reply:
Yes, thanks.